Documentation/ApiOverview/Authentication/CSRFlikeRequestTokenHandling.rst

```diff
@@ -65,6 +65,10 @@ The sequence looks like the following:
     objects (later created implicitly in this example) are organized in the
     :php:`\TYPO3\CMS\Core\Context\SecurityAspect`.
 
+    ..  versionchanged:: 13.3
```

Comment:
Why is this mentioned here?

Reply:
Because the code snippet uses the new factory and ppl might be confused when they stumble on that